<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutFre</class>
 <widget class="QDialog" name="AjoutFre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>204</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Ajout de plages de fréquences</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Borne inférieure</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>251</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>3</number>
    </property>
    <item>
     <widget class="QTextEdit" name="plage1">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit::indicator{
	width:90px;
	height: 50px;

}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Borne supérieure</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="spacing">
     <number>3</number>
    </property>
    <item>
     <widget class="QTextEdit" name="plage2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>29</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit::indicator{
	width:90px;
	height: 50px;

}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="submitPlages">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>192</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Nombre de fréquences souhaités</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>91</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <property name="spacing">
     <number>3</number>
    </property>
    <item>
     <widget class="QTextEdit" name="nb_plages">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit::indicator{
	width:90px;
	height: 50px;

}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_11">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Ajout d'une antenne</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_12">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_13">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_13">
    <item>
     <widget class="QTextEdit" name="Name_Ant">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>29</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_14">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>150</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_14">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Position en X</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_15">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>190</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_15">
    <item>
     <widget class="QTextEdit" name="Pos_AntX">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>240</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Position en Y</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_16">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>270</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_16">
    <item>
     <widget class="QTextEdit" name="Pos_AntY">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="BtnValAnt">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Charger un fichier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnLoadFichier">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>470</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Charger</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_17">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>470</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_17">
    <item>
     <widget class="QTextEdit" name="NomFichier">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
